spec/test_apps/runtime_options/dumpers/response_dumper_helper.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+RailsResponseDumper.merge_options({
+                                    exclude_response_headers: true
+                                  })
```

Comment:
This formats nicely on one line:

```suggestion
RailsResponseDumper.merge_options({ exclude_response_headers: true })
```